Python/pywarpx/picmi.py

```diff
@@ -737,10 +743,12 @@ def initialize_inputs(self):
         self.solver.initialize_inputs()
 
         for i in range(len(self.species)):
-            self.species[i].initialize_inputs(self.layouts[i],
-                                              self.initialize_self_fields[i],
-                                              self.injection_plane_positions[i],
-                                              self.injection_plane_normal_vectors[i])
+            self.species[i].initialize_inputs(
+                self.layouts[i],
+                False if self.initialize_self_fields[i] is None else True,
```

Comment:
Without digging too deeply into the code, I'm slightly worried on reading this that it messes with the potential case of `self.initialize_self_fields[i] == False`

Should the whole argument expression instead read

`self.initialize_self_fields[i] is not None and self.initialize_self_fields[i]`